web/src/app/pages/main-page-container/main-page/main-page.component.html

```diff
@@ -20,12 +20,13 @@
   <div class="content">
     <mat-sidenav-container class="main-page-container">
       <mat-sidenav-content class="mat-sidenav-content">
+        <submission-detail-panel *ngIf="showSubmissionPanel"
+          [submission]="selectedSubmission"
+          (closePanel)="closeSelectedSubmissionDetailPanel()">
+        </submission-detail-panel>
         <ground-map
           [shouldEnableDrawingTools]="shouldEnableDrawingTools"
         ></ground-map>
-        <div *ngIf="shouldEnableDrawingTools">
```

Comment:
Is there a reason to delete this?

I believe `shouldEnableDrawingTools` is currently set to `false` because it's temporary disabled and this shouldn't affect others.

Reply:
Added it back!